compute_endpoint/globus_compute_endpoint/engines/globus_compute.py

```diff
@@ -23,11 +23,21 @@
 from parsl.multiprocessing import SpawnQueue
 
 logger = logging.getLogger(__name__)
-DOCKER_CMD_TEMPLATE = "docker run {options} -v {rundir}:{rundir} -t {image} {command}"
-APPTAINER_CMD_TEMPLATE = "apptainer run {options} {image} {command}"
-SINGULARITY_CMD_TEMPLATE = "singularity run {options} {image} {command}"
-PODMAN_CMD_TEMPLATE = "podman run {options} -v {rundir}:{rundir} -t {image} {command}"
-VALID_CONTAINER_TYPES = ("docker", "singularity", "apptainer", "podman", "custom", None)
+# Docker, podman, and podman-hpc all use the same syntax
+DOCKER_CMD_TEMPLATE = (
+    "{cmd_name} run {options} -v {rundir}:{rundir} -t {image} {command}"
+)
+# Apptainer and Singularity use the same syntax
+APPTAINER_CMD_TEMPLATE = "{cmd_name} run {options} {image} {command}"
```

Comment:
(Can avoid the `black` overflow to 3 lines if you like; consider `{cmd}` or `{exe}` as alternative choices that I argue would be just as clear.  But in parenthesis because not a blocker, just a suggestion.)